proteus/tests/test_sedclosure.py

```diff
@@ -0,0 +1,102 @@
+from proteus import Comm, Profiling
+import numpy as np
+import numpy.testing as npt
+import unittest
+
+comm = Comm.init()
+Profiling.procID = comm.rank()
+
+Profiling.logEvent("Testing SedClosure")
+
+
+class TestHsu(unittest.TestCase):
+    def testGradularDrag1(self):
+        from proteus.mprans.SedClosure import HsuSedStress
+        # Constant params
+        aDarcy = 1.
+        bForch = 1.
+        grain = 0.1
+        packFraction = 0.2
+        packMargin = 0.01
+        uf = np.array([5,4])
```

Comment:
@adimako I think the issue may be that uf and us are arrays of integers here but are  getting passed  to C++ as double arrays. You can either explicitly give  the dtype  as 'd' in the array() function or make it 5.,4. etc.  There's probably a way to be more specific in Cython to enforce the dtype of the ndarray's,